pic-create

```diff
@@ -23,8 +23,13 @@ this_dir=`dirname $0`
 
 #only copy submit and params if we clone from default pic folder
 default_folder_to_copy="submit include/simulation_defines/param include/simulation_defines/unitless"
-#if we clone a project we copy full include 
-folder_to_clone="submit include"
+#if we clone a project we copy full include
+if [ -d "$1/pymodules" ] ; then
```

Comment:
REQUIRED:
The code did not check yet, if `$1` is set, this is done in line 53. Please move this code after line 53 and perhaps add `pymodules` there **or* check the positional arguments here.